### DISCUSSION TITLE
Hardware connectivity to desk

### DISCUSSION THREAD

**Discussion by matthewtraughber** [developit/desky#2] [Q&A] [answered]
@developit - I'm trying to create something similar to this in order to allow simultaneous control of multiple desks.

It's unclear to me how you interfaced the ESP8266 you linked from Amazon to the actual desk (as it only has micro USB). I know the OEM control boards use a 10P10C connector with only 4 actual wires used, so what cable/connector did you end up using for this?

If it's not painfully obvious, this will be my first foray into development boards such as Arduino/Raspberry Pi, etc :)

Thanks!

**Comment by developit** [comment, marked as answer]
Hi @matthewtraughber

I bought a cable that had a 10p10c connector from Amazon for $5 (I believe it is from an NCR POS/cash register), then stripped the wires corresponding to the 4 wires used in the connector on the desk motor control box. Those connect directly to 4 pins/holes on the WeMos D1mini 5V (power/+), G (ground/-), RX and TX (communication receive/send):

![PXL_20220219_180221764.MP~2.jpg](https://user-images.githubusercontent.com/105127/154813260-df549a81-f7fa-4536-b344-046290d1072d.jpg)

Happy to video chat if you're confused where to start - it took some trial and error for me too, I hadn't done this kind of stuff for years.

**Reply by matthewtraughber** [reply]
Thanks for the update & picture - it's very helpful!

It looks like we're unclear what the 5th wire (SLEEP) is used for, but good to know it isn't required to get this working.

I'm having issues finding a similar cable on Amazon (RJ50 / 10P10C) that isn't $15+ each, so I may end up grabbing some 10P10C male ends and repurposing old cat5e I have lying around.

Appreciate the video chat offer; let me get all the hardware together and possibly fry a few boards before I take up any more of your time. :)
